actions/kpops-runner/action.yaml

```diff
@@ -0,0 +1,46 @@
+name: "Kpops runner"
+description: "Run Kpops command line with the given params"
+
+inputs:
+  mode:
+    description: "The command mode you want to use to launch kpops"
+    required: true
+  pipeline:
+    description: "Pipeline file you want to run"
+    required: true
+  working-directory:
+    description: "The root directory containing the config.yaml, pipelines folder and defaults"
+    default: "."
+  execute:
+    description: "Set execution flag"
+    default: "false"
+  defaults:
+    description: "Defaults folder path"
+    default: "defaults"
+  config:
+    description: "Config.yaml file path"
+    default: "config.yaml"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out repository code
+      uses: actions/checkout@v3
+      
+    - run: |
+        pip install kpops==0.2.0.dev20221122085523 --extra-index-url https://test.pypi.org/simple/
+      name: Install Kpops
+      shell: bash
```

Comment:
Now these 2 lines need to go before `run`, otherwise they are bash code

Reply:
Since it works its minor, but I would still prefer to have the name and the shell before the `run`

Reply:
Since it works its minor, but I would still prefer to have the name and the shell before the `run`

Reply:
now ?

Reply:
Yes, thanks :)